pkg/config/security/security.go

```diff
@@ -94,25 +93,29 @@ func ValidateAttribute(key string, values []string) error {
 	case hasPrefix(key, attrRequestHeader):
 		return validateMapKey(key)
 	case isEqual(key, attrSrcIP):
-		return validateIPs(values)
+		return ValidateIPs(values)
 	case isEqual(key, attrSrcNamespace):
-	case isEqual(key, attrSrcUser):
 	case isEqual(key, attrSrcPrincipal):
 	case isEqual(key, attrRequestPrincipal):
 	case isEqual(key, attrRequestAudiences):
 	case isEqual(key, attrRequestPresenter):
 	case hasPrefix(key, attrRequestClaims):
 		return validateMapKey(key)
 	case isEqual(key, attrDestIP):
-		return validateIPs(values)
+		return ValidateIPs(values)
 	case isEqual(key, attrDestPort):
-		return validatePorts(values)
-	case hasPrefix(key, attrDestLabel) || isEqual(key, attrDestName, attrDestNamespace, attrDestUser):
-		return fmt.Errorf("deprecated attribute (%s): only supported in v1alpha1", key)
+		return ValidatePorts(values)
```

Comment:
NIT: Some validate APIs start with upper case V and some start with lower case "v". Can we be consistent?

Reply:
The functions with upper case V are exported to external package and used in the `validation.go`. The functions with lower case v is only used in this file so I think we should keep it  until it's needed in other packages.